src/main/java/com/epam/task/first/logic/ArrayLogic.java

```diff
@@ -0,0 +1,112 @@
+package com.epam.task.first.logic;
+
+import com.epam.task.first.entity.Array;
+
+import java.util.ArrayList;
+
+public class ArrayLogic {
+    private Array array;
```

Comment:
your class should be stateless (don't keep this as field, pass in in method as argument)